qt/TorrentFilter.cc

```diff
@@ -218,7 +220,14 @@ bool TorrentFilter::lessThan(QModelIndex const& left, QModelIndex const& right)
 
 bool TorrentFilter::trackerFilterAcceptsTorrent(Torrent const* tor, QString const& tracker) const
 {
-    return tracker.isEmpty() || tor->hasTrackerSubstring(tracker);
+    if (tracker == TRACKERLESS_FILTER)
```

Comment:
Comparing translatable (`TRACKERLESS_FILTER`) and non-translatable (`tracker`) strings is a bad idea (as you have already mentioned yourself).